play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StandaloneAhcWSRequest.scala

```diff
@@ -151,10 +154,9 @@ case class StandaloneAhcWSRequest(
   }
 
   private def withBodyAndContentType(wsBody: WSBody, contentType: String): Self = {
-    if (headers.contains(HttpHeaders.Names.CONTENT_TYPE)) {
-      copy(body = wsBody)
-    } else {
-      copy(body = wsBody).addHttpHeaders(HttpHeaders.Names.CONTENT_TYPE -> contentType)
+    this.contentType match {
+      case Some(_) => copy(body = wsBody)
+      case None => copy(body = wsBody).withContentType(contentType)
```

Comment:
I think that keeping the first write makes more sense since using the `Content-Type` from the body is not very explicit in the public API. So, in the code below:

```scala
req.withContentType("multipart/form-data")
   .withBody("something something")
```
As a user, I would rather prefer that`withContentType` wins.

WDYT?